tests/ctst/steps/iam-policies/common.ts

```diff
@@ -0,0 +1,217 @@
+        },
+        {
+            API: 'GetObjectTagging',
+            checkResponse: this.getObjectTaggingResponseCode,
+        },
+        {
+            API: 'PutObjectTagging',
+            checkResponse: this.putObjectTaggingResponseCode,
+        },
+        {
+            API: 'DeleteObjectVersion',
+            checkResponse: this.deleteObjectResponseCode,
+        },
+        {
+            API: 'GetBucketReplication',
+            checkResponse: this.getReplicationConfigurationResponseCode,
+        },
+        {
+            API: 'GetBucketLifecycle',
+            checkResponse: this.getLifecycleConfigurationResponseCode,
+        },
+        {
+            API: 'PutBucketLifecycle',
+            checkResponse: this.putLifecycleConfigurationResponseCode,
+        },
+        {
+            API: 'PutBucketReplication',
+            checkResponse: this.putReplicationConfigurationResponseCode,
+        },
+        {
+            API: 'GetObjectVersion',
+            checkResponse: this.getObjectResponseCode,
+        },
+        {
+            API: 'GetObjectVersionRetention',
+            checkResponse: this.getBucketObjectRetentionResponseCode,
+        },
+        {
+            API: 'PutObjectVersionRetention',
+            checkResponse: this.putObjectRetentionResponseCode,
+        },
+        {
+            API: 'GetObjectVersionLegalHold',
+            checkResponse: this.getObjectLegalHoldResponseCode,
+        },
+        {
+            API: 'PutObjectVersionLegalHold',
+            checkResponse: this.putObjectLegalHoldTaggingResponseCode,
+        },
+        {
+            API: 'GetObjectVersionTagging',
+            checkResponse: this.getObjectTaggingResponseCode,
+        },
+        {
+            API: 'DeleteObjectVersionTagging',
+            checkResponse: this.deleteObjectVersionTaggingResponseCode,
+        },
+        {
+            API: 'PutObjectVersionTagging',
+            checkResponse: this.putObjectTaggingResponseCode,
+        },
+        {
+            API: 'GetObjectVersionAcl',
+            checkResponse: this.getObjectAclResponseCode,
+        },
+        {
+            API: 'PutObjectVersionAcl',
+            checkResponse: this.putObjectAclResponseCode,
+        },
+        {
+            API: 'GetBucketTagging',
+            checkResponse: this.getBucketTaggingResponseCode,
+        },
+        {
+            API: 'PutBucketTagging',
+            checkResponse: this.putBucketTaggingResponseCode,
+        },
+        {
+            API: 'DeleteBucketTagging',
+            checkResponse: this.deleteBucketTaggingResponseCode,
+        },
+    ];
+    
+    let userCredentials;
+    if ([EntityType.IAM_USER, EntityType.ACCOUNT].includes(this.saved.type)) {
+        userCredentials = this.parameters.IAMSession;
+    } else {
+        userCredentials = this.parameters.AssumedSession;
+    }
+    const api = testAPIs.find(api => api.API === action);
```

Comment:
let's raise an error if api is not found in the list

Reply:
https://github.com/scality/Zenko/pull/1809/files#diff-284f11d6681baa8186a18ff40a18801ef2f353d3d23a6d8be5d52e3367ed92c9R180

this ok?